tldextract/tldextract.py

```diff
@@ -210,7 +208,7 @@ def __call__(self, url):
 
         if not tld and netloc and netloc[0].isdigit():
             try:
-                is_ip = socket.inet_aton(netloc)
```

Comment:
Yay cleaning up unused variables! Boo this block getting more confusing what the side-effects are doing. I'll extract this into a function.